src/wiz_l7.c

```diff
@@ -124,3 +126,57 @@ DEFINE_DO_FUN (do_force) {
     }
     send_to_char ("Ok.\n\r", ch);
 }
+
+/*
+ * Confiscates an object from a player.  The command is useful for cases when
+ * an immortal needs to get a noremove or nodrop item from a player to do
+ * a restring or if they need to confiscate something for really any other
+ * reason.
+ *
+ * Code by Keridan of Benevolent Iniquity, additions and conversions for
```

Comment:
@blakepell This is a bit paranoid, but do you know if there are any licensing concerns with this? I'm assuming not, the only code I could find from Keridan were under no license: http://www.mudmagic.com/codes/download/diku/merc/rom/snippets/sacall-keri.txt

Reply:
Sorry, I miss notifications on here sometimes.  To my knowledge there is no license or I would have noted that to myself.  I believe the link you identified is from the same individual.

In this case to, it would be hard to code a command to do this and do it much differently which is to say, it's a very basic command that builds off of logic that already exists in your mud.